configure.ac

```diff
@@ -82,6 +82,13 @@ AS_IF([test -z "$PANDOC"],
     [AC_MSG_WARN([Required executable pandoc not found, man pages will not be built])])
 AM_CONDITIONAL([HAVE_PANDOC],[test -n "$PANDOC"])
 
+AC_ARG_ENABLE([integration],
+            [AS_HELP_STRING([--enable-integration],
+                            [build integration tests against TPM (default is no)])],
+            [enable_integration=$enableval],
+            [enable_integration=no])
+AM_CONDITIONAL([INTEGRATION], [test "x$enable_integration" != xno])
+
```

Comment:
Since we have `--enable-integration` now, we can remove [`PKG_CHECK_MODULES([OATH],[liboath])`](https://github.com/tpm2-software/tpm2-totp/blob/8ea64587212f78e7bdb2666e6c6993005d8e325f/configure.ac#L78) in favour of something like
```
AS_IF([test "x$enable_integration" != xno], [PKG_CHECK_MODULES([OATH],[liboath])])
```
to require liboath only if the test suite is enabled.

Reply:
Done